caffe2/perfkernels/batch_box_cox_Sve128.cc

```diff
@@ -0,0 +1,241 @@
+#if defined(__aarch64__) && defined(CAFFE2_PERF_WITH_SVE128)
+#include <arm_neon.h>
+#include <arm_neon_sve_bridge.h>
+#include <arm_sve.h>
+
+// Log and exp approximations inspired from ACL implementation
+
+inline float32x4_t vtaylor_polyq_for_log_f32(float32x4_t x) {
+  const float32x4_t log_tab_1 = vdupq_n_f32(-2.29561495781f);
+  const float32x4_t log_tab_2 = vdupq_n_f32(-2.47071170807f);
+  const float32x4_t log_tab_3 = vdupq_n_f32(-5.68692588806f);
+  const float32x4_t log_tab_4 = vdupq_n_f32(-0.165253549814f);
+  const float32x4_t log_tab_5 = vdupq_n_f32(5.17591238022f);
+  const float32x4_t log_tab_6 = vdupq_n_f32(0.844007015228f);
+  const float32x4_t log_tab_7 = vdupq_n_f32(4.58445882797f);
+  const float32x4_t log_tab_8 = vdupq_n_f32(0.0141278216615f);
+
+  float32x4_t A = vmlaq_f32(log_tab_1, log_tab_5, x);
```

Comment:
Please use `auto` to avoid potential casting errors (both clang and gcc will silently typecast NEON return to incorrect vectorized type, but will error out if its passed as an argument)
```suggestion
  const auto A = vmlaq_f32(log_tab_1, log_tab_5, x);
```